projects/step-core/src/lib/components/edit-scheduler-task-dialog/edit-scheduler-task-dialog.component.html

```diff
@@ -0,0 +1,61 @@
+<step-modal-window [title]="isNew ? 'New task' : 'Edit task'">
+  <div step-modal-window-body>
+    <step-alert *ngIf="error" type="danger">
+      {{ error }}
+    </step-alert>
+    <form #formContainer="ngForm">
+      <step-form-field>
+        <step-label>Name</step-label>
+        <input type="text" placeholder="Name" required [(ngModel)]="_task.attributes!['name']" name="name" />
+      </step-form-field>
+      <step-select-plan required [ngModel]="plan" (ngModelChange)="handlePlanChange($event)" name="plan">
+      </step-select-plan>
+      <step-cron-selector
+        label="CRON Expression"
+        placeholder="CRON Expression"
+        required
+        [(ngModel)]="_task.cronExpression"
+        name="cron"
+      >
+      </step-cron-selector>
+      <step-custom-forms
+        [stModel]="_task.executionsParameters!.customParameters!"
+        (stModelChange)="handleCustomParametersChange($event)"
+        stScreen="executionParameters"
+      >
+      </step-custom-forms>
+      <step-form-field>
+        <step-label>User</step-label>
+        <input
+          type="text"
+          placeholder="User"
+          required
+          [ngModel]="_task.executionsParameters!.userID"
+          (ngModelChange)="handleUserIdChange($event)"
+          name="user"
+        />
+      </step-form-field>
+    </form>
+    <mat-expansion-panel [(expanded)]="showParameters">
+      <mat-expansion-panel-header>
+        <mat-panel-title>Advanced</mat-panel-title>
+      </mat-expansion-panel-header>
+      <section class="advanced-body">
+        <step-form-field>
+          <step-label>Parameters</step-label>
+          <textarea
```

Comment:
I think this should use a reactive form and have a JSON validator which would then display an error below if the JSON is invalid (I've done that as part of the function configuration dialog migration)

Reply:
@bojidaryovchev This editor uses `step-custom-froms` component, that is downgraded, is used in legacy UI and doesn't support reactive forms. That's why I decided to use template driven form for this dialog.

Reply:
@dvladir I also used this component in the function configuration dialog

Reply:
![image](https://github.com/exense/step-frontend/assets/8954749/ed57e346-3dee-485d-8c11-fa4ec9c2363d)

Reply:
![image](https://github.com/exense/step-frontend/assets/8954749/31eab148-6a0c-43d2-bad8-a93e6f176b33)

Reply:
![image](https://github.com/exense/step-frontend/assets/8954749/3a08d2bb-d06c-4eda-8903-dd4b54c4daf6)

Reply:
![image](https://github.com/exense/step-frontend/assets/8954749/b4838697-4e8b-4c12-8827-ede627a1f7c5)